source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +123,124 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
 * dhcpv6-stateful
 * dhcpv6-stateless
 
-Given the priorities stated in [Objectives](#objectives), the ovirt-provider-ovn should only accept the 'dhcpv6-stateful' configuration, and fail for the other 2 options. When not provided, this parameter will of course default to the only accepted value.
+Given the priorities stated in [Objectives](#objectives), the ovirt-provider-ovn
+will accept either the 'dhcpv6-stateful' or 'dhcpv6-stateless' configuration,
+while *slaac* is not yet supported. When not provided, this parameter will
+**default to dhcpv6-stateless**, since it allows for a simpler guest VM
+configuration.
+
```

Comment:
Having a look at the supported use cases per dhcpv6 configuration, I think dhcpv6-stateful is more suitable for our users.
@mburman5 @MosheSheena What is your point of view?

Reply:
I think both stateful and stateless are relevant for our users.

Reply:
But which of them should be the default value, which is the only possible value for subnets created by oVirt webadmin (if we do not decide to add choice to webadmin)?

Reply:
I think the default should be stateless, since stateful only works out of the box in a subset of the targeted OSs - centOS 7 .

stateless implies more pieces, but works for the whole set.

Should I refer that in the feat page?

Reply:
> I think the default should be stateless, since stateful only works out of the box in a subset of the targeted OSs - centOS 7 .
> 

Isn't stateful working out of the box, too?
https://github.com/oVirt/ovirt-site/pull/1951#discussion_r268554772

Reply:
*with* router-

Reply:
> _with_ router-

this applies to stateless, too

Reply:
@dominikholler the default value today when creating the subnet via the engine UI is - 
"ipv6_address_mode": "dhcpv6_stateful",

Reply:
>  the default value today when creating the subnet via the engine UI is -
"ipv6_address_mode": "dhcpv6_stateful"
@mburman5 Do yo think we should keep it?

@maiqueb Do you see any reason to change it?

Reply:
@dominikholler Today when creating subnet via the engine UI, there is no default value for 
ipv6_address_mode":
this value doesn't appear when crated via the UI. 
I'm not sure what should be the default, it depend on what be used more.

Reply:
we're using stateful, since that's the only available implementation.

@dominikholler backwards compatibility wins, since I can't find another argument for changing it to stateless.

Reply:
ok